internal/uploader/dummy.go

```diff
@@ -0,0 +1,26 @@
+package uploader
+
+import (
+	"context"
+
+	"github.com/go-openapi/strfmt"
+	eventsapi "github.com/openshift/assisted-service/internal/events/api"
+	"github.com/sirupsen/logrus"
+)
+
+type DummyUploader struct {
```

Comment:
Same here, not sure this type needs to be exported.

Reply:
Gotcha, modified it to a private type!